package.json

```diff
@@ -20,13 +20,19 @@
 		"@types/prompts": "^2.4.2",
 		"@types/tar": "^6.1.4",
 		"@types/yargs": "^17.0.22",
+		"@typescript-eslint/eslint-plugin": "^5.59.2",
+		"@typescript-eslint/parser": "^5.59.2",
 		"esbuild": "^0.17.6",
 		"esbuild-plugin-node-externals": "^1.0.1",
+		"eslint": "^8.39.0",
+		"eslint-plugin-prettier": "^4.2.1",
 		"pkg": "^5.8.0",
+		"prettier": "^2.8.8",
 		"ts-node": "^10.9.1",
 		"typescript": "^4.9.5"
 	},
 	"dependencies": {
+		"@speakeasy-sdks/hc-sdk": "^1.3.1",
```

Comment:
Noticed there's a `generate: sdk-client` command above that uses the `openapi-generator-cli`. Is it important to maintain this?

We'd be able to generate from our CLI (ex. `speakeasy generate sdk -s openapi.yaml -o sdk-client -l typescript`), just haven't made that switch yet. Instead, we're declaring the generated SDK as a dependency